projects/components/src/load-async/loader/loader.component.ts

```diff
@@ -1,15 +1,38 @@
-import { ChangeDetectionStrategy, Component } from '@angular/core';
-import { IconType } from '@hypertrace/assets-library';
-import { IconSize } from '../../icon/icon-size';
+import { ChangeDetectionStrategy, Component, Input, OnChanges } from '@angular/core';
+import { ImagesAssetPath, LoaderTypes } from '@hypertrace/assets-library';
+import { assertUnreachable } from '@hypertrace/common';
 
 @Component({
   selector: 'ht-loader',
   styleUrls: ['./loader.component.scss'],
   template: `
     <div class="ht-loader">
-      <ht-icon icon="${IconType.Loading}" size="${IconSize.Large}"></ht-icon>
+      <img [ngClass]="[this.type]" [src]="this.imagePath" />
```

Comment:
So, as per my other comment, Please have both ht-icon and img here. If loader type is RowExpandable or Page then we use Img tag. If not, we can use the existing ht-icon. So, let's ng-if on this condition.

Reply:
https://github.com/hypertrace/hypertrace-ui/pull/1071#discussion_r697169599